paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -254,6 +273,13 @@
                     },
                     "additionalProperties": true
                 },
+                "marathon_shard": {
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "previous_marathon_shards": {
+                    "type": "array"
+                },
```

Comment:
`k8s_shard`?

Reply:
Right, we also need to modify this later.